etc/completions

```diff
@@ -923,4 +923,4 @@ $opt
 # fallbacks
 # =========
 
-$command
+$command~fallback $opt* ($revision|$path|$remote)* -- $path*
```

Comment:
Question for reviewers: What do you think of the `$variable~modifier` syntax I came up with for this?

Reply:
🤷‍♂️ 

Two thoughts, really:

I'd say that it's not obvious what it means.
And the `modifier` name also makes me think of BEM, which would use a `--` instead.

Reply:
Following a suggestion from @sharplet I've made the modifier a prefix rather than a suffix. `$command~fallback` has become `fallback::$command`

The double colon (`::`) is to get around the fact that a single colon (`:`) appears in various places in the existing tab completion rules, and recognising the difference without complicating the parser code would be really tricky.

I think this is an improvement, but I'm open to more feedback on it.